cli/command/run/cmd.go

```diff
@@ -849,3 +852,20 @@ func awaitGlobalTimeout() {
 		os.Exit(1)
 	}
 }
+
+// checkForUpdates check if there is a saucectl update available.
+func checkForUpdates() {
+	gh := github.Client{
+		HTTPClient: &http.Client{Timeout: githubTimeout},
+		URL:        "https://api.github.com",
+	}
+
+	v, err := gh.HasUpdateAvailable()
+	if err != nil {
+		log.Error().Err(err).Msg("Check for update")
```

Comment:
Didn't we want failures to be silent?

PS: You may want to change the failure message here.

Reply:
My bad, I forgot to staged this during my last commit.